src/pages/ceph/users/index.vue

```diff
@@ -1,47 +1,67 @@
 <script setup lang="ts">
 import { ConnectError } from '@connectrpc/connect';
-import { useCephStore } from '~/store/ceph';
 
+import { ref } from 'vue';
+import { Dialog, DialogPanel, DialogTitle, TransitionChild, TransitionRoot } from '@headlessui/vue';
+import { XMarkIcon } from '@heroicons/vue/24/outline';
+import { useCephStore } from '~/store/ceph';
 useHead({
-    title: 'Ceph Users',
+    title: 'MGR Dashboard Users',
 });
 definePageMeta({
-    title: 'Ceph Users',
+    title: 'MGR Dashboard Users',
     requiresAuth: true,
 });
 
+const modalWindowOpen = ref(false);
+const username = ref();
+
 const { $grpc } = useNuxtApp();
 const cephStore = useCephStore();
 
 const { data: cephUsers } = useLazyAsyncData('cephUsers', async () => {
     try {
-        const users = await cephStore.getCephUsers();
-        console.log(users);
-        return users;
+        return await cephStore.getCephUsers();
     } catch (e) {
         $grpc.handleError(e as ConnectError);
     }
 });
+
+const displayConfirmation = function (Username: string) {
+    modalWindowOpen.value = true;
+    username.value = Username;
+    console.log(Username);
+};
```

Comment:
I would recommend using https://vueuse.org/core/useConfirmDialog/ if you have time to look into it